test/twin/default_test.rb

```diff
@@ -8,7 +8,7 @@ class Twin < Disposable::Twin
     feature Default
 
     property :title, default: "Medio-Core"
-    property :genre, default: -> { "Punk Rock #{model.class}" }
+    property :genre, default: ->(*) { "Punk Rock #{model.class}" }
```

Comment:
Oh, do we need to add this to the signature?

Reply:
Yeah, because we pass `**keyword_arguments`. It should, right ?

Reply:
@apotonick This has been fixed with `representable-3.1.1` 🍻